package.json

```diff
@@ -11,6 +11,7 @@
     "build:types": "lerna run types",
     "build": "yarn webfont:build && yarn build:core && yarn build:packages && yarn build:types && yarn build:docs",
     "check:audit": "/bin/bash -c 'yarn audit --level critical; [[ $? -ge 16 ]] && exit 1 || exit 0'",
+    "check:deps": "lerna exec --ignore docs --no-bail --stream -- depcheck",
```

Comment:
maybe we should exclude the `packages/**/tests` folder and only include the `packages/**/src` folder? 🤔

Reply:
Ben, pour moi c'est important que les (dev)dependencies des tests sont inclus (listé / installé) aussi. J'ai rajouté `"@testing-library/*"` dans le `depcheckrc.json` parce que c'est inclus a la racine.